### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by valeriosouza** [dd32/Github-to-WordPress-Plugins-Sync#2]
Sorry if this is a dumb question, but how to use this script?

**Comment by davidperezgar** [comment]
Yes, I've this doubt as well. It's a php, that is supposed to be in a server? and called via webhook from github? I've tested like that, but it seems that it does not work...
